code/modules/nifsoft/nif.dm

```diff
@@ -293,7 +297,7 @@ You can also set the stat of a NIF to NIF_TEMPFAIL without any issues to disable
 			stat = NIF_TEMPFAIL
 			return FALSE
 
-	var/percent_done = (world.time - (install_done - (35 MINUTES))) / (35 MINUTES)
+	var/percent_done = (world.time - (install_done - (15 MINUTES))) / (15 MINUTES)
```

Comment:
```suggestion
	var/percent_done = (world.time - (install_done - (15 MINUTES))) / (15 MINUTES) // CHOMPedit: Install time from 35 minutes to 15 minutes.
```
Chompedit every edited line if not block-commenting

Reply:
Oops, thought I did that haha